src/example-relay/src/components/Button.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import * as React from 'react';
+import * as sx from '@adeira/sx';
+
+type Props = {|
+  +onClick?: () => void,
+  +children: React.Node,
+  +dataTest?: string,
+  +disabled?: boolean,
+  +iconLeft?: React.Node,
+  +iconRight?: React.Node,
+  +type?: 'submit' | 'button',
+|};
+
+export default function Button({
+  children,
+  dataTest,
+  iconLeft,
+  iconRight,
+  type = 'button',
+  ...rest
+}: Props): React.Node {
+  return (
+    <button
+      // eslint-disable-next-line react/button-has-type
+      type={type}
+      data-testid={dataTest}
+      className={styles('button', 'primary', rest.disabled ? 'disabled' : 'button')} // TODO: Allow conditional style in sx
```

Comment:
This should be already possible in the latest SX version. Could you please update this code? Or is there still some problem we need to fix?

Reply:
No, I forgot to fix it after rebasing, thanks for pointing it out 😊

Reply:
🚀